_events/2022-All-Hands-Meeting.md

```diff
@@ -15,7 +15,7 @@ header_credit: '"Field House w/ Water Tower" by Majdan is licensed under CC BY-N
 
 {% capture main %}
 
-Save the dates! The OSG All-Hands Meeting 2022 will be 14–18 March 2022. We are planning a hybrid event:
+Registration is now open for the OSG All-Hands Meeting 2022 will be 14–18 March 2022. Complete registration details are [here](https://opensciencegrid.org/all-hands/2022/registration/).  We are planning a hybrid event:
```

Comment:
Please read these after making changes! This sentence makes no sense now. Probably something like “Registration is now open for the OSG All-Hands Meeting 2022, which will be held 14–18 March 2022.”